.envrc

```diff
@@ -0,0 +1 @@
+use flake
```

Comment:
Probably should be removed from this/added to `.gitignore`

Reply:
Similarly, the `flake.[nix|lock]` should also be removed and probably added to the `.gitignore`